src/Kunstmaan/ApiBundle/Model/ApiPagePart.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ruud
+ * Date: 23/10/2016
+ * Time: 19:54
+ */
+
+namespace Kunstmaan\ApiBundle\Model;
+
+
+use Kunstmaan\PagePartBundle\Entity\AbstractPagePart;
+
+class ApiPagePart
+{
+    /** @var string */
+    private $type;
+
+    /** @var AbstractPagePart */
+    private $pagePart;
+
+    /** @var string */
+    private $context;
+
+    /**
+     * @return string
+     */
+    public function getType()
+    {
+        return $this->type;
+    }
+
+    /**
+     * @param string $type
+     */
+    public function setType($type)
+    {
+        $this->type = $type;
+    }
+
+    /**
+     * @return AbstractPagePart
+     */
+    public function getPagePart()
+    {
+        return $this->pagePart;
+    }
+
+    /**
+     * @param AbstractPagePart $pagePart
+     */
+    public function setPagePart(AbstractPagePart $pagePart)
```

Comment:
You shouldn’t typehint for `AbstractPagePart`, use `PagePartInterface` instead

Reply:
done